keyboards/linworks/fave60/config.h

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2020 Moritz Plattner
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+/* number of backlight levels */
+#define BACKLIGHT_PIN B7
+#define BACKLIGHT_LEVELS 3
+#define BACKLIGHT_BREATHING
+#define BACKLIGHT_ON_STATE 1
+
+#define RGB_DI_PIN F5
```

Comment:
This will need to be updated to `ws2812.pin` for develop.  
```suggestion
#define WS2812_DI_PIN F5
#define RGB_DI_PIN WS2812_DI_PIN
```